config.md

```diff
@@ -384,31 +384,80 @@ For POSIX platforms, the configuration structure supports `hooks` for configurin
         * **`env`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2008's `environ`][ieee-1003.1-2008-xbd-c8.1].
         * **`timeout`** (int, OPTIONAL) is the number of seconds before aborting the hook.
             If set, `timeout` MUST be greater than zero.
-    * **`poststart`** (array of objects, OPTIONAL) is an array of [post-start hooks](#poststart).
-        Entries in the array have the same schema as pre-start entries.
-    * **`poststop`** (array of objects, OPTIONAL) is an array of [post-stop hooks](#poststop).
-        Entries in the array have the same schema as pre-start entries.
+    * **`create-runtime`** (array of objects, OPTIONAL) is an array of [`create-runtime` hooks](#create-runtime-hooks).
+        Entries in the array have the same schema as `prestart` entries.
```

Comment:
I realized while reading the rendered [`config.json`](https://github.com/opencontainers/runtime-spec/blob/master/config.md) that:
* the spec for the new hooks shouldn't build on the structure of a deprecated hook
* the spec for the new hooks _cannot_ reuse the structure verbatim because the definition for **`path`** explicitly says:
    >  Runtimes MUST resolve this value in the [runtime namespace](glossary.md#runtime-namespace).
  
  which - AFAICT - is contradictory to the requirement of `create-container` and `start-container` to be run in the container namespace

Reply:
Good catch!
Though, I think for **`path`**, we could lookup the binary in the runtime namespace (and should?) for the `create-container` hook.